docs/_templates/layout.html

```diff
@@ -260,6 +312,23 @@
 
   {%- block footer %} {% endblock %}
 
+  <!-- Load and configure Algolia DocSearch. -->
+  <script
+    type="text/javascript"
+    src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"
+  ></script>
+  <script type="text/javascript">
+    docsearch({
+      apiKey: "c533a6bb7a6fd2e5d424daaeac242abb",
+      indexName: "streamlit",
+      inputSelector: "#docsearch-input",
+      debug: false, // Set debug to true if you want to inspect the dropdown
+      algoliaOptions: {
+        // TODO: pass in different versions depending on what docs version we're on
+        'facetFilters': ["version:latest"]
```

Comment:
@randyzwitch Do you know if there's an easy way to grab the version from the docs? Ideally, we replace "latest" which whatever version the user is currently on (eg "stable" or "0.74.1" or something)

Reply:
Not sure. I'm sure you can, as it is available at build time as `{READTHEDOCS_VERSION}`

https://github.com/streamlit/streamlit/blob/develop/docs/env_setup.sh#L24